meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -31,6 +31,9 @@ SDKMACHINE ?= "x86_64"
 # Generate source mirror tarballs:
 BB_GENERATE_MIRROR_TARBALLS = "1"
 
+# Do not allow building of anything that hasn't been checked beforehand.
+SUPPORTED_RECIPES_CHECK = "fatal"
+
 # Configure images in "development mode".
```

Comment:
we still need control on which recipes and from which layers got built, especially as we are dependant on so many upstream layers with conflicting recipes. We need to verify list of what we are getting as result with current master build before merging this PR